drivers/adc/ad9656/ad9656.c

```diff
@@ -0,0 +1,248 @@
+/***************************************************************************//**
```

Comment:
the changelog was added in the commit description. The commit description should contain some information about the driver some basic info about the part not fixes that were done after the review process.

Reply:
Changed the commit description to contain a brief description of the AD9656 chip.